neuron/src/lib/Neuron/Zettelkasten/Query.hs

```diff
@@ -111,3 +118,5 @@ graphQueryResultJson q r skippedZettels =
     resultJson = case q of
       GraphQuery_Id ->
         toJSON r
+      GraphQuery_BacklinksOf _ _ ->
+        toJSON r
```

Comment:
Can you paste the JSON output this generates?

Reply:
```
$ ./result/bin/neuron -d guide query --backlinks-of 4a6b25f1
```

```json
{
  "skipped": {},
  "result": [
    [
      "Folgezettel",
      {
        "zettelTags": [
          "walkthrough"
        ],
        "zettelDay": "2020-03-19",
        "zettelID": "2011406",
        "zettelError": {
          "Right": []
        },
        "zettelContent": [],
        "zettelQueries": [
          [
            "ZettelQuery_ZettelByID",
            [
              "2011403",
              "OrdinaryConnection"
            ]
          ],
          [
            "ZettelQuery_ZettelByID",
            [
              "2012301",
              null
            ]
          ],
          [
            "ZettelQuery_ZettelByID",
            [
              "2011504",
              "OrdinaryConnection"
            ]
          ],
          [
            "ZettelQuery_ZettelByID",
            [
              "2013501",
              "OrdinaryConnection"
            ]
          ],
          [
            "ZettelQuery_ZettelByID",
            [
              "4a6b25f1",
              null
            ]
          ]
        ],
        "zettelTitle": "Creating and Editing zettels",
        "zettelTitleInBody": true
      }
    ],
    [
      "OrdinaryConnection",
      {
        "zettelTags": [],
        "zettelDay": null,
        "zettelID": "6f0f0bcc",
        "zettelError": {
          "Right": []
        },
        "zettelContent": [],
        "zettelQueries": [
          [
            "ZettelQuery_ZettelByID",
            [
              "4a6b25f1",
              "OrdinaryConnection"
            ]
          ]
        ],
        "zettelTitle": "Philosophy",
        "zettelTitleInBody": true
      }
    ]
  ],
  "query": [
    "GraphQuery_BacklinksOf",
    [
      null,
      "4a6b25f1"
    ]
  ]
}
```

```
$ ./result/bin/neuron -d guide query --uplinks-of 4a6b25f1 | jq
```

```json
{
  "skipped": {},
  "result": [
    [
      "Folgezettel",
      {
        "zettelTags": [
          "walkthrough"
        ],
        "zettelDay": "2020-03-19",
        "zettelID": "2011406",
        "zettelError": {
          "Right": []
        },
        "zettelContent": [],
        "zettelQueries": [
          [
            "ZettelQuery_ZettelByID",
            [
              "2011403",
              "OrdinaryConnection"
            ]
          ],
          [
            "ZettelQuery_ZettelByID",
            [
              "2012301",
              null
            ]
          ],
          [
            "ZettelQuery_ZettelByID",
            [
              "2011504",
              "OrdinaryConnection"
            ]
          ],
          [
            "ZettelQuery_ZettelByID",
            [
              "2013501",
              "OrdinaryConnection"
            ]
          ],
          [
            "ZettelQuery_ZettelByID",
            [
              "4a6b25f1",
              null
            ]
          ]
        ],
        "zettelTitle": "Creating and Editing zettels",
        "zettelTitleInBody": true
      }
    ]
  ],
  "query": [
    "GraphQuery_BacklinksOf",
    [
      "Folgezettel",
      "4a6b25f1"
    ]
  ]
}
```